site/docs/projection.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: docs
+menu: docs
+title: Projection
+permalink: /docs/projection.html
+---
+## Projection Overview
+Projection's are specified at the mark specification level, alongside encoding.
+
+{:#properties}
+## Projection Properties
+{% include table.html props="type,clipAngle,clipExtent,center,rotate,precision" source="Projection" %}
+
+
+*Note*: All [properties](#properties) of projections are **optional** with defaults as defined in the [Vega projection properties](https://vega.github.io/vega/docs/projections/#properties). Because of this, defined marks that don't have explicitly defined projections may implicitly derive a projection. Implicit projections will be added for any [`geoshape`](geoshape.html) mark or any mark that has fields of type [`latitude`](type.html#latitude), [`longitude`](type.html#longitude), or [`geojson`](type.html#geojson).
```

Comment:
> All [properties](#properties) of projections are **optional** with defaults as defined in the [Vega projection properties]

Can you just copy all default values and add them to the JSDocs in the interface (reduce the need for users to jump to Vega docs, which can be quite confusing as it's totally a different language.)


> defined marks that don't have explicitly defined projections may implicitly derive a projection. 

Unclear what is "defined marks"?